CMakeLists.txt

```diff
@@ -645,6 +645,12 @@ endif(WITH_EXAMPLES)
 # them tests
 #
 
+# First fuzzing driver test
```

Comment:
Yeah, but should move down after `WITH_TESTS` or maybe even `UNIX` down below?

Reply:
I can do that.
But could we have fuzzing without unit tests ?
What do you prefer ?

Reply:
I always try to stick to "When in Rome, do as Romans do" - so how do other projects handle this?

Reply:
I think most do have different compile options to enable fuzzing, tests, examples/programs.

Do you want me to move fuzzing after tests ? (beyond them)

Reply:
Please move it within the `WITH_TESTS` section so we have unit tests, integration tests and fuzzing tests under one umbrella for the time being. It might make sense to split this up later on, but I'd like to defer this decision.